flink-connector-gcp-pubsub/pom.xml

```diff
@@ -34,9 +34,9 @@ under the License.
 
 	<packaging>jar</packaging>
 
-	<properties>
-		<google-cloud-libraries-bom.version>8.1.0</google-cloud-libraries-bom.version>
-	</properties>
+<!--	<properties>-->
+<!--		<google-cloud-libraries-bom.version>8.1.0</google-cloud-libraries-bom.version>-->
+<!--	</properties>-->
```

Comment:
If we don't need this lines for v3.0, let's remove them instead of commenting them out

Reply:
updated